spyrmsd/rmsd.py

```diff
@@ -257,15 +267,16 @@ def symmrmsd(
     """
 
     if isinstance(coords, list):  # Multiple RMSD calculations
-        RMSD: Any = []
+        RMSD: List[float] = []
```

Comment:
```suggestion
        RMSD: Any = []
```
If you revert this change (or using `Union[float, List[flost]]`, can you get around the `mypy` issues about re-definint RMSD and avoid defining `single_RMSD` and having an early return in one of the branches?